tests/validators/arguments_v3/test_general.py

```diff
@@ -0,0 +1,149 @@
+import pytest
+
+from pydantic_core import ArgsKwargs, SchemaError, SchemaValidator, ValidationError
+from pydantic_core import core_schema as cs
+
+from ...conftest import Err, PyAndJson, plain_repr
+
+
+@pytest.mark.parametrize(
+    ['input_value', 'expected'],
+    (
+        [ArgsKwargs(()), ((), {})],
+        [{}, ((), {})],
+        [ArgsKwargs((1,)), Err('', [{'type': 'unexpected_positional_argument'}])],
+        [ArgsKwargs((), {'a': 1}), Err('', [{'type': 'unexpected_keyword_argument'}])],
+        # TODO?
+        # [{'a': 1}, Err('', [{'type': 'unexpected_argument'}])],
+    ),
+)
+def test_no_args(py_and_json: PyAndJson, input_value, expected) -> None:
```

Comment:
The current validation logic currently does not forbid extra mapping items, if they aren't consumed to validate against a parameter. Should we do so? It is doable, but a bit more cumbersome to implement I think.

Reply:
i.e. the following is currently allowed:

```python
v = SchemaValidator(cs.arguments_v3_schema([]))
v.validate_python({'extra': 1})
```

Reply:
we should definitely support (by default) forbidding extra arguments.

Reply:
I now realize we try to read extra behavior from the schema, and default from the config (both for the v3 and current arguments schema). I'll add an extra schema key allowing forbid and ignore, and default to forbid.